llama-index-integrations/embeddings/llama-index-embeddings-cohere/llama_index/embeddings/cohere/base.py

```diff
@@ -284,3 +333,24 @@ def _get_text_embeddings(self, texts: List[str]) -> List[List[float]]:
     async def _aget_text_embeddings(self, texts: List[str]) -> List[List[float]]:
         """Get text embeddings."""
         return await self._aembed(texts, input_type="search_document")
+
+    def _get_image_embedding(self, img_file_path: str) -> List[float]:
```

Comment:
The input to this function is of type `ImageFileType` -- which can be string or bytesIO

Reply:
Sure. I will change it accordingly.